spec/controllers/api/events_controller_spec.rb

```diff
@@ -18,7 +18,7 @@ def overridden_params(new_params)
       let(:workflow) { create(:workflow) }
       let(:project) do
         p = workflow.project
-        p.update_columns(migrated: true, configuration: { zoo_home_project_id: zoo_home_project_id })
+        p.update_columns(migrated: true, id: zoo_home_project_id, configuration: { zoo_home_project_id: zoo_home_project_id })
```

Comment:
Because the UPP is initialized in the events controller with `project_id: legacy_zoo_project.id`. If this and the project id below aren't changed, then many of the specs will fail due to foreign key restraints on upp.project_id not referencing existing projects.